server/etcdserver/api/membership/storev2.go

```diff
@@ -94,7 +94,7 @@ func mustSaveMemberToStore(lg *zap.Logger, s v2store.Store, m *Member) {
 }
 
 func mustDeleteMemberFromStore(lg *zap.Logger, s v2store.Store, id types.ID) {
-	if _, err := s.Delete(MemberStoreKey(id), true, true); err != nil {
+	if _, err := s.Delete(MemberStoreKey(id), true, true); err != nil && !isKeyNotFound(err) {
```

Comment:
Don't change this function. The function above `RemoveMember` assumes `isKeyNotFound` causes panic.

Reply:
It isn't a big problem.

- Actually it doesn't matter. Even time when applying conf change, etcd will validate the conf change(search `ValidateConfigurationChange`). It will never run into panic. It's exactly the reason why we never see panic (bug) before
- We will remove the code of adding or removing members into v2store in following PRs soon.

Reply:
@geetasg Please revert this change, and raise a separate PR, so as not be blocked on this non-important thing.